instrumentation/finatra-2.9/javaagent/finatra-2.9-javaagent.gradle

```diff
@@ -34,11 +34,19 @@ dependencies {
     // Requires old version of Jackson
     testImplementation enforcedPlatform("com.fasterxml.jackson:jackson-bom:2.9.10")
   }
-  testImplementation "com.twitter:finatra-http_2.11:19.12.0"
+  testImplementation("com.twitter:finatra-http_2.11:19.12.0") {
+    // Finatra POM references linux-aarch64 version of this which we don't need. Including it
+    // prevents us from managing Netty version because the classifier name changed to linux-aarch_64
+    // in recent releases. So we exclude and force the linux-x86_64 classifier instead.
+    exclude group: "io.netty", module: "netty-transport-native-epoll"
+  }
+  testImplementation "io.netty:netty-transport-native-epoll:4.1.51.Final:linux-x86_64"
```

Comment:
Will this break tests on ARM processors? We certainly will have to support them some time in the future

Reply:
It shouldn't break tests, it will fallback to non-epoll if the OS doesn't match. We currently have something similar here

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/119697b1315db6bcf108fcf314bb0e95af7d6d4f/instrumentation/netty/netty-4.1/javaagent/netty-4.1-javaagent.gradle#L38

And could add `aarch_64` to both projects later if needed.

Reply:
looks like muzzle is failing related to this

```
1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':instrumentation:finatra-2.9:javaagent:muzzle-AssertPass-com.twitter-finatra-http_2.11-21.2.0'.
> Could not resolve all files for configuration ':instrumentation:finatra-2.9:javaagent:muzzle-AssertPass-com.twitter-finatra-http_2.11-21.2.0'.
   > Could not find netty-transport-native-epoll-4.1.65.Final-linux-aarch64.jar (io.netty:netty-transport-native-epoll:4.1.65.Final).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/io/netty/netty-transport-native-epoll/4.1.65.Final/netty-transport-native-epoll-4.1.65.Final-linux-aarch64.jar

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':instrumentation:finatra-2.9:javaagent:muzzle-AssertPass-com.twitter-finatra-http_2.12-21.2.0'.
> Could not resolve all files for configuration ':instrumentation:finatra-2.9:javaagent:muzzle-AssertPass-com.twitter-finatra-http_2.12-21.2.0'.
   > Could not find netty-transport-native-epoll-4.1.65.Final-linux-aarch64.jar (io.netty:netty-transport-native-epoll:4.1.65.Final).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/io/netty/netty-transport-native-epoll/4.1.65.Final/netty-transport-native-epoll-4.1.65.Final-linux-aarch64.jar
```

Reply:
Yup was good motivation to port in that DD muzzle enhancement :)